httpbin/main.go

```diff
@@ -0,0 +1,20 @@
+package main
```

Comment:
can you please move this file to cmd/httpbin? It's the idiomatic source tree structure in Go.